src/subscription-engine.ts

```diff
@@ -1,302 +0,0 @@
-/* jslint node: true */
-"use strict";
-
-import { logger } from "@dojot/dojot-module-logger";
-import kafka = require("kafka-node");
-import util = require("util");
-import { KafkaConsumer } from "./consumer";
-import { KafkaFactory } from "./KafkaFactory";
-import { KafkaProducer } from "./producer";
-import tools = require("./simple-tools");
-
-import { Event } from "./subscription/Event";
-import { Notification } from "./subscription/Notification";
```

Comment:
Does subscription-engine was deprecated? If yes, you can put some comments telling that was deprecated and why.